### PR TITLE
Rajesh/makescriptgeneric

### PR DESCRIPTION
**Pull request by Rajesh2015**
-Added misc changes to make script generic 
-Refactoring some part due to path

### CODE REVIEW

**Comment by Rajesh2015** [comment]
@henrikbulldog Please do review

File: README.md [hunk 1]
```diff
@@ -37,24 +37,19 @@ deleteJarsTask := {
 ```
 - All your settings can be put inside a config.yaml file like below(Applicable only for shell script).
 ```
-ORGANIZATION : <orgnaziation docker hub name>
```
Comment (henrikbulldog) [inline README.md:40]: Not valid YAML, missing quotes for some strings, quotes used for integers, comma missing in object rules, spaces missing after : ,
inconsistent casing of vars , VERSION should be version
should be formatted like this:

```
dockerusername: "zeidkleo"
version: "latest"
databricks_profile: "dev"
roles: {
  instance_profile_arn: "arn:aws:iam::853628181016:instance-profile/ETL_Databricks_Development_Role",
  assume_role_arn: "",
}
zone_id: "eu-west1b"
min_workers: 2
max_workers: 8
node_type_id: "i3.2xlarge"
autotermination_minutes: 120
driver_node_type_id: "i3.2xlarge"

```

**Review by henrikbulldog — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rajesh2015) [reply README.md:40]: Will do required change and remove version

Comment (Rajesh2015) [reply README.md:40]: Donot merge this one yet .I will do required changes here

Comment (Rajesh2015) [reply README.md:40]: Done all required changes and added optimisation and now check status of each command before proceeding ahead

File: README.md [hunk 2]
```diff
@@ -49,12 +46,9 @@ roles: {
 zone_id: "<enter region>"
 min_workers: "<min workers>"
```
Comment (henrikbulldog) [inline README.md]: Remove quotes

File: README.md [hunk 3]
```diff
@@ -49,12 +46,9 @@ roles: {
...
 max_workers: "<max workers>"
```
Comment (henrikbulldog) [inline README.md]: Remove quotes

File: README.md [hunk 4]
```diff
@@ -49,12 +46,9 @@ roles: {
...
-spark_version: "<spark version in cluster>"
-node_type_id : "<node type>"
+node_type_id: "<node type>"
 autotermination_minutes: "<time out>"
```
Comment (henrikbulldog) [inline README.md]: Remove quotes

File: README.md [hunk 5]
```diff
@@ -37,10 +37,7 @@ deleteJarsTask := {
...
 - All your settings can be put inside a config.yaml file like below(Applicable only for shell script).
 ```
-ORGANIZATION : <orgnaziation docker hub name>
-REPOSITORY_NAME : <repositort name>
-VERSION : <version of repository>
-clustername: "<enter cluster name>"
+dockerusername: "<docker hub user name>"
 databricks_profile: "<enter profile>"
 roles: {
  instance_profile_arn: "<enter role>"
```
Comment (henrikbulldog) [inline README.md]: Add comma

**Review by henrikbulldog — CHANGES_REQUESTED** [review]
There seems to be a problem with naming jar:
image

Got error on cluster:
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.ClassFormatError: Absent Code attribute in method that is not native or abstract in class file com/databricks/rpc/JsonSerializable

File: README.md (hunk 4 above)
Comment (Rajesh2015) [reply README.md]: Thanks for catching This seems to be missed though code changes was done

**Comment by Rajesh2015** [comment]
> There seems to be a problem with naming jar:
> image
> 
> Got error on cluster:
> Error: A JNI error has occurred, please check your installation and try again
> Exception in thread "main" java.lang.ClassFormatError: Absent Code attribute in method that is not native or abstract in class file com/databricks/rpc/JsonSerializable

Strange thing is this jar comes with the confg-version.jar .In script we don't rename anything .So that should be ok if its getting copied to docker container .Not sure about the other JNI error

**Comment by Rajesh2015** [comment]
As discussed all the string will be inside quote and number will not have nay quote

**Pull request merged**